Extension/src/commands.ts

```diff
@@ -44,10 +45,18 @@ class TemporaryCommandRegistrar {
 
     public registerTempCommand(command: string): void {
         this.tempCommands.push(vscode.commands.registerCommand(command, () => {
+            if (this.isDisabled) {
+                vscode.window.showInformationMessage("The C/C++ command is disabled due to C_Cpp.intelliSenseEngine set to Disabled.");
```

Comment:
"This command is disabled when \"C_Cpp.intelliSenseEngine\" is set to \"Disabled\""